profile_service/src/main/kotlin/ru/zveron/mapper/LotsMapper.kt

```diff
@@ -0,0 +1,22 @@
+package ru.zveron.mapper
+
+import ru.zveron.contract.profile.LotStatus
+import ru.zveron.contract.profile.LotSummary
+import ru.zveron.contract.lot.model.Lot
+import ru.zveron.contract.profile.lotSummary
+
+object LotsMapper {
+
+    fun List<Lot>.toBuilder(status: LotStatus): Collection<LotSummary> =
```

Comment:
```suggestion
    fun List<Lot>.toBuilder(status: LotStatus): List<LotSummary> =
```
Хотя наверное непринципиально